spec.html

```diff
@@ -26064,7 +26682,8 @@ <h1>
                 1. Append _ee_ to _starExportEntries_.
               1. Else,
                 1. Append _ee_ to _indirectExportEntries_.
-            1. Return Source Text Module Record { [[Realm]]: _realm_, [[Environment]]: *undefined*, [[Namespace]]: *undefined*, [[Status]]: ~unlinked~, [[EvaluationError]]: *undefined*, [[HostDefined]]: _hostDefined_, [[ECMAScriptCode]]: _body_, [[Context]]: ~empty~, [[ImportMeta]]: ~empty~, [[RequestedModules]]: _requestedModules_, [[ImportEntries]]: _importEntries_, [[LocalExportEntries]]: _localExportEntries_, [[IndirectExportEntries]]: _indirectExportEntries_, [[StarExportEntries]]: _starExportEntries_, [[DFSIndex]]: *undefined*, [[DFSAncestorIndex]]: *undefined* }.
+            1. Let _async_ be _body_ Contains `await`.
```

Comment:
Seems like a worthwhile shorthand to define. I'm leaning towards allow this here and elsewhere.

Reply:
> I see that we do something similar with "Contains `super`" in the Early Errors for script and module bodies, so I guess there's precedent.

E.g. ```It is a Syntax Error if |ModuleItemList| Contains `super`.``` But that's an example of omitting the `is *true*`, so isn't a precedent for the usage here. I.e.
```Let _async_ be _body_ Contains `await`.```
*isn't* an abbreviation of:
```Let _async_ be _body_ Contains `await` is *true*.```

---

If you're talking about a Contains-invocation supplying the value for a Let, then the only precedent is in the default definition for Contains:
```Let _contained_ be the result of _child_ Contains _symbol_.```
I think `the result of` is there to prevent a garden-path parse. E.g., if it were
```Let _contained_ be _child_ Contains _symbol_.```
then someone might read
```Let _contained_ be _child_```
as a unit, and then be confused when they hit `Contains`. But if you think there's a low risk of confusion, then dropping `the result of` is okay. (Most Contains-invocations don't have it.)